src/common/themes/peggysPastels.js

```diff
@@ -2,6 +2,7 @@
  * https://www.deviantart.com/tpenguinltg/art/Peggy-s-Pastels-505540096
  */
 
+// eslint-disable-next-line @typescript-eslint/no-var-requires
```

Comment:
I don't think this is actually working, `../utils/index.js` is not exported as a file on the package relative to the themes export:

https://www.runpkg.com/?react95@3.12.1/dist/

Reply:
Great catch, I tried importing this theme in this sandbox and it indeed is not working: https://codesandbox.io/s/react95-template-urwo3s?file=/src/index.js

Reply:
fixed it in #289